client/my-sites/stats/feedback/modal/index.tsx

```diff
@@ -58,9 +74,20 @@ const FeedbackModal: React.FC< ModalProps > = ( { siteId, onClose } ) => {
 				} )
 			);
 
+			disableFeedbackSubmissionInOneDay().then( () => {
```

Comment:
This name makes me think it will be disabled after 24 hours instead of for the next 24 hours. I'd recommend `disableFeedbackSubmissionForOneDay()` instead.

Reply:
Addressed in [3aa11f3](https://github.com/Automattic/wp-calypso/pull/94197/commits/3aa11f38e0c321ec325b286b1cb50243245c8d0b).